### ISSUE TITLE
a11y: heading levels

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#779]
**What Should Happen**
Every page should have a single `<h1>` element.  Where other headings are present, they should be in sequential order (i.e. `<h2>`, then `<h3>`, etc.) and properly nested, according to the structure of the page / information present.

Headings should be used only for section titles, to help provide a sense of the high-level summary of the page (think: table of contents).  Other content should not use headings, but should rather use CSS styles for visual emphasis.

**JIRA Issues**
- [x] MOVE-185
- [x] MOVE-184
- [x] MOVE-172
- [x] MOVE-171
- [x] MOVE-169
- [x] MOVE-155
- [x] MOVE-143
- [x] MOVE-130
- [x] MOVE-123
- [x] MOVE-110
- [x] MOVE-106
- [x] MOVE-105
- [x] MOVE-91
- [x] MOVE-54 (use headings instead of `<header>`)
- [x] MOVE-53
- [x] MOVE-27
- [x] MOVE-21
- [x] MOVE-20
- [x] MOVE-11
- [x] MOVE-4
- [x] for modals, use `<h2>` instead of `<h1>` for modal titles
